### PR TITLE
timeliney:0.1.0

### PR DESCRIPTION
**Pull request by pta2002**
I am submitting
- [ ] a new package
- [x] an update for a package

What's changed:
- Bumped CeTZ to 0.2.2
- Added `offset` parameter

### CODE REVIEW

File: packages/preview/timeliney/0.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Timeliney
+
+Create Gantt charts automatically with Typst!
+
+Here's a fully-featured example:
+
+```typst
+#import "@preview/timeliney:0.0.1"
```
Comment (elegaanz) [inline packages/preview/timeliney/0.1.0/README.md:8]:
```suggestion
#import "@preview/timeliney:0.1.0"
```

File: packages/preview/timeliney/0.1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      ])
+    )
+  }
+)
+```
+
+![Example Gantt chart](sample.png)
+
+## Installation
+Import with `#import "@preview/timeliney:0.0.1"`. Then, call the `timeliney.timeline` function.
```
Comment (elegaanz) [inline packages/preview/timeliney/0.1.0/README.md:61]:
```suggestion
Import with `#import "@preview/timeliney:0.1.0"`. Then, call the `timeliney.timeline` function.
```

**Review by elegaanz — COMMENTED** [review]
Just two outdated version numbers in the README, LGTM otherwise :)

**Comment by elegaanz** [comment]
Also, CeTZ was just updated to 0.3.0, maybe you want to update to that version before I merge? Otherwise it can be done in a future release too.

File: packages/preview/timeliney/0.1.0/README.md (hunk 1 above)
Comment (pta2002) [reply packages/preview/timeliney/0.1.0/README.md:8]: Oh nice catch, thanks!

**Comment by pta2002** [comment]
I will check out the CeTZ version later, if it works as-is I'll update it, otherwise I'll put it in a new version.

**Comment by pta2002** [comment]
Hi @elegaanz, from me we can merge this, I'll work on the CeTZ update in a later version

**Comment by elegaanz** [comment]
Okay :)

**Pull request merged**